spec/features/view_peeps_spec.rb

```diff
@@ -0,0 +1,17 @@
+feature 'Viewing peeps' do
+  scenario 'Page displays time peep was peeped' do
+    Peep.create(content: 'This is my second peep!')
+    visit('/peeps')
+    within 'ul#peeps' do
+      time = Time.now
+      time = time.strftime("%H:%M")
+      expect(page).to have_content 'This is my second peep!'
+      expect(page).to have_content time
+    end
+  end
+  scenario 'Username is dislayed with peep' do
```

Comment:
Try to use whitespace idiomatically.  The scenarios should be separated by a blank line.